### ISSUE TITLE
Program bővítési sorrend

### ISSUE THREAD

**Issue by jack298** [fulopm/itsh-szotar-java#4]
Alap: Grafikus felület + Bemenet + Online szótárban keresés

1. Bővítés:
Excelben való keresés 
HA NINCS
online szótárban keresés
(excel fájl beolvasása, feldolgozása,, keresés megírása) 

2. Bővítés: 
Excelben való keresés 
HA NINCS 
online szótárban való keresés VAGY Excelhez hozzáadás 
(Excelbe írás, Grafikus bemenet elkészítése)

3. Bővítés: 
Excelben való keresés LEHET szerkeszteni, törölni 
HA NINCS
online szótárban való keresés VAGY Excelhez hozzáadás
(Excel sorainak módosítása, fájdalommentes törlése)

4. bővítés:
játékok..

**Comment by fulopm** [comment]
Elkezdjük akkor csinálni a GUI-t? Csak a menüket létrehozni, stb, és akkor utána kiegészítgetjük mindig?

**Comment by fulopm** [comment]
GUI-hoz úgy sem kell sok tervezés, adatstruktúrák stb

**Comment by jack298** [comment]
@fulopm Igen, a GUI-t, meg az online keresést, az sem nagy kihivás, viszont ad egy skeleton programot, ahonnan el lehet indulni

**Comment by fulopm** [comment]
OK. Akkor elkezdem a GUI-t, szépen modulárisan felépítve. Az összes többi kiosztott feladat egyenlőre ne élvezzen prioritást, csak próbálkozzanak. Pl a Benji feladata egyenlőre maradjon meg tanulásnak (de attól még csinálja, mert később használni fogjuk tudni!). Ha meglesz a GUI által alkotott skeleton, akkor szépen tudjuk mögétenni az egyes funkciókat. @BenjiVok @dsaltcutter
